config/management/src/genesis.rs

```diff
@@ -19,7 +21,7 @@ use vm_genesis::ValidatorRegistration;
 #[derive(Debug, StructOpt)]
 pub struct Genesis {
     #[structopt(flatten)]
-    pub backend: SingleBackend,
+    pub backend: SharedBackend,
```

Comment:
nit: I'd consider renaming "backend" -> "shared_backend" to be consistent and explicit? (e.g., here, in verify() and layout()?)

Reply:
there's only one backend, so it seemed excessive, I'll see if Michael notices or complains ;)